server/package.json

```diff
@@ -0,0 +1,20 @@
+{
+  "name": "screeps-server",
+  "private": true,
+  "version": "1.0.0",
+  "main": "index.js",
+  "repository": "https://github.com/Jomik/screeps-server",
+  "author": "Jonas Holst Damtoft <jonasdamtoft@gmail.com>",
+  "license": "MIT",
+  "engines": {
+    "npm": "~6",
+    "node": "10 - 12"
+  },
+  "dependencies": {
+    "screeps": "4.2.19",
+    "js-yaml": "4.1.0"
+  },
+  "scripts": {
+    "cli": "docker compose exec screeps cli"
```

Comment:
```suggestion
```
Let's not have this - we don't use this package.json for anything other than building the image